src/scripts/main.js

```diff
@@ -1,3 +1,12 @@
 'use strict';
 
 // write code here
+
+const spider = document.querySelector('.spider');
+const wall = document.querySelector('.wall');
+
+const heigth1 = wall.getBoundingClientRect().height / 2 - 50;
+const weigth1 = wall.getBoundingClientRect().width / 2 - 50;
```

Comment:
There is a typo in the variable name 'weigth1'. It should be 'width1' to correctly represent the width.